doc/storage.md

```diff
@@ -10,11 +10,14 @@ discourse: 7735
 About storage </explanation/storage>
 Create a storage pool <howto/storage_create_pool>
 Create an instance in a pool <howto/storage_create_instance>
-Add a storage volume <howto/storage_add_volume>
-Configure storage settings <howto/storage_configure>
-List pools and volumes <howto/storage_list>
-Move or copy volumes <howto/storage_move>
-Back up storage volumes <howto/storage_backup>
-Resize storage <howto/storage_resize>
+Create a storage volume <howto/storage_create_volume>
```

Comment:
@ru-fu was there a specific order you wanted for this list?

Reply:
I was thinking do it hierarchically, so cover storage pools first, and then volumes?

Reply:
I think the current order is fine.